src/utilities/activities.ts

```diff
@@ -73,6 +73,18 @@ export function activitiesToPoints(
     .map(activity => activityToPoint(activitiesMap, activity, selectedActivityId));
 }
 
+/**
+ * Transforms activities to tags.
+ * Sorts tags in alphabetical order.
+ */
+export function activitiesToTags(activities: Activity[]): string[] {
```

Comment:
Not required but I'm wondering if we should just have this function defined inline the derived store. Do you think we will reuse it?

Reply:
Unsure if we'll reuse this, happy to inline it.

Reply:
Though I suppose this allows us to write a test for this activity -> tags functionality.

Reply:
Is it straightforward to test the derived store directly?

Reply:
Yeah probably not too hard, just need to populate the store before the test which I now know how to do actually. Just not sure exactly which test to put it in.

Reply:
You should be able to just use the `set` function on the store I think. This file might have some usage inspiration: https://github.com/sveltejs/svelte/blob/master/test/store/index.ts

Reply:
Inlined it.